package/MDAnalysis/lib/util.py

```diff
@@ -1274,3 +1275,23 @@ def blocks_of(a, n, m):
                    a.strides[0], a.strides[1])
 
     return np.lib.stride_tricks.as_strided(a, new_shape, new_strides)
+
+
+class Namespace(object):
```

Comment:
The class would benefit from a `__getitem__` method.

For now, I can add an auxiliary field with a invalid name, and I loose all access to it:

```python
universe.trajectory.add_auxiliary('an invalid name', 'test.xvg', represent_ts_as='closest')
# No error message, but how do I get the values?
```

A `__getitem__` method would allow me to access the auxiliary like this:

```python
universe.trajectory.aux['an invalid name']
```